elephant/test/test_spike_train_correlation.py

```diff
@@ -579,7 +579,7 @@ def setUp(self):
 
     def test_sttc(self):
         # test for result
-        target = 0.8748350567
+        target = 0.495860165593
```

Comment:
Why did the target change?

Reply:
The previous target was wrong due to the unit error in the calculation. I ran the original C code which this function is based on; it yields 0.4958602 which is consistent with my result. [Here](https://onlinegdb.com/HkIK9PeTX) is the ready-to-run C code with the same input parameters in an online compiler.

Reply:
I can confirm this after checking the units.